agents/smtp.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+from .agent import ServerAgent
+
+
+class SMTPAgent(ServerAgent):
+
+    def __init__(self, logfile):
+        super(SMTPAgent, self).__init__(logfile)
+
+        self.port = 25
+
+    def service_healthy(self):
+        try:
+            output = subprocess.Popen(
+                ["ps", "aux"], stdout=subprocess.PIPE
+            ).communicate()[0]
+
+            return (
+                self.port_open()
+                and any(
+                    process in output
+                    for process in ["postfix", "exim", "sendmail"]
```

Comment:
Add a `self.smtp_processes` list so subclasses or configs can customize the list:
```
self.smtp_processes = ["postfix", "exim", "sendmail"]
...
any(proc in output for proc in self.smtp_processes)
```

Reply:
Addressed this by making `smtp_processes` into an attribute which can currently be set at construction of `SMTPAgent`. I don't know whether in the future we will be configuring agents in some specific manner, so for now it seems like a viable solution.